test/Microsoft.ML.TimeSeries.Tests/TimeSeriesEstimatorTests.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.TimeSeriesProcessing;
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests
+{
+    public class TimeSeriesEstimatorTests : TestDataPipeBase
+    {
+        private const int inputSize = 150528;
+
+        private class Data
+        {
+            public float Value;
+
+            public Data(float value)
+            {
+                Value = value;
+            }
+        }
+
+        private class TestDataXY
+        {
+            [VectorType(inputSize)]
+            public float[] A;
+        }
+        private class TestDataDifferntType
+        {
+            [VectorType(inputSize)]
+            public string[] data_0;
+        }
+
+        public TimeSeriesEstimatorTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        void TestSsaChangePointEstimator()
```

Comment:
So...

![image](https://user-images.githubusercontent.com/8295757/47742659-55c52f00-dc3a-11e8-956d-31ab1ebf0968.png)

That's a *long* time for a test to run, and this isn't even on the slower build machines, but on my dev box here, which is comparatively fast. Could you tone it down a bit?